README.md

```diff
@@ -21,112 +29,166 @@ If you think you've found a bug or would like to suggest an enhancement, please
+The Salt master needs to resolve all its Salt minions by their host names, as well as all Salt minions need to resolve the Salt master by its host name. 
+
+Configure, enable, and start the NTP time synchronization server on all nodes:
+- `systemctl enable ntpd.service`
+- `systemctl start ntpd.service`
+
+Check whether the AppArmor service is running and disable it on each cluster node to avoid problems.
+- `zypper disable apparmor.service`
+
+Check whether the Firewall service is running and disable it on each cluster node to avoid problems.
+- `systemctl disable SuSEfirewall2.service`
+
+Note that the Cluster deployment with DeepSea will not work with Firewall/AppArmor enabled.
+
+Install DeepSea on the node which will be the Salt master:
+- `zypper in deepsea`
+
+The command installs the salt-master and salt-minion packages as a dependency as well as Grafana/Prometheus
+
+Check that the salt-master service is enabled and started - enable and start it if needed:
+- `systemctl enable salt-master.service`
+- `systemctl start salt-master.service`
+
+Install the package salt-minion on all minion nodes.
+- `zypper in salt-minion`
+
+Configure all minions (including the master minion) to connect to the master. If your Salt master is not reachable by the DNS name salt, 
+edit the file "/etc/salt/minion" or create a new file "/etc/salt/minion.d/master.conf" with the following content:
+- `master: DNS_name_of_salt_master`
+
+If you performed any changes to the configuration files mentioned above, restart the Salt service on all Salt minions:
+- `systemctl restart salt-minion.service`
+
+Check that the file /srv/pillar/ceph/master_minion.sls on the Salt master points to your Salt master. If your Salt master is reachable via 
+more host names, use the one suitable for the storage cluster. If you used the default host name for your Salt master in the example domain, then the file looks as follows:
+- `master_minion: salt.example`
+
+Check that the salt-minion service is enabled and started on all nodes. Enable and start it if needed:
+- `systemctl enable salt-minion.service`
+- `systemctl start salt-minion.service`
+
+Accept all salt keys on the Salt master:
+- `salt-key --accept-all`
+
+Verify that the keys have been accepted:
+- `salt-key --list-all`
+
+#### Cleanup your Disks - only needed if your disks were used in a cluster before
+
+Prior to deploying a cluster with DeepSea make sure that all disks that were used as OSD by previous clusters are empty without partitions. To ensure this, you have to manually zap all the disks. 
+Remember to replace 'X' with the correct disk letter - Attention - This command will remove all your data from a disk, so please be careful:
+
+Wipe the beginning of each partition:
+
+`for partition in /dev/sdX[0-9]
+do
+  dd if=/dev/zero of=$partition bs=4096 count=1 oflag=direct
+done`
+
+Wipe the partition table:
+- `sgdisk -Z --clear -g /dev/sdX`
+
+Wipe the backup partition tables:
+- `size=`blockdev --getsz /dev/sdX``
```

Comment:
the quotes interfer witht he formatting here. Maybe use other quotes, use escaping or block quotes for the code block?